spec/plsql/procedure_spec.rb

```diff
@@ -2251,3 +2251,49 @@ def new_candidate(status)
   end
 
 end
+
+describe '#get_argument_metadata' do
+  before do
```

Comment:
I would vote for being explicit about the [`before`/`after` scope](https://www.relishapp.com/rspec/rspec-core/v/3-4/docs/hooks/before-and-after-hooks).